### ISSUE TITLE
Table mapping doesn't work on Typo3 9.5.19

### ISSUE THREAD

**Issue by Beltshassar** [sourcebroker/t3api#8]
Table mapping doesn't seem to work with T3api on Typo3 9.5.19

I've tried with my own extension and t3apinews. In both cases I get the response: "Table XYZ doesn't exist" when I'm using the api.
Typo3 backend-records works as expected.

    {
      "hydra:title": "Table 'kas_imh.tx_t3apinews_domain_model_news' doesn't exist",`
      "hydra:code": 1472074485,
      "hydra:description": "Table 'kas_imh.tx_t3apinews_domain_model_news' doesn't exist"
    }

Right now I'm trying my luck with Typo3 10.4 wich seems to work.

**Comment by klamparski** [comment]
@Beltshassar I just tested it with t3api `1.0.5`, t3apinews `0.3.2` and TYPO3 `9.5.19` - works perfectly fine for me.
Could you please ensure that mapping configuration is read correctly in TypoScript Object Browser in BE?

**Comment by Beltshassar** [comment]
The Object Browser looks like this - which seems okay afiak
![image](https://user-images.githubusercontent.com/15835725/85406284-dcdabf80-b561-11ea-88ea-3f2a5bb94c50.png)

sourcebroker/t3api: 1.0.5
sourcebroker/t3apinews: 0.3.2
PHP: 7.2.26

I'm using composer to set up Typo3 and install extensions, could that affect the autoloader?
Same server as the working Typo3 10.4 instance.
Anyway, it works as intended on 10.4, so I will continue development there.

**Comment by klamparski** [comment]
@Beltshassar You were right about that issue. I just faced the same one and found the bug. 

It was caused by not precise order of request middlewares.

The funny thing is that I couldn't reproduce it before because it worked correctly when TYPO3 core's `adminpanel` extension wa activated. `adminpanel` registers its own request middlewares and this registration affects the order in a way that t3api request was run correctly.

Issue is fixed within commit https://github.com/sourcebroker/t3api/commit/001e8fc69e5a22639c7602d59d73d7d47e12f161.
